Events/Ch28.event

```diff
@@ -0,0 +1,247 @@
+#include EAstdlib.event
+
+EventPointerTable(0x7F,ThisChapter)
+
+
+ThisChapter:
+POIN TurnBasedEvents
+POIN CharacterBasedEvents
+POIN LocationBasedEvents
+POIN MiscBasedEvents
+POIN Dunno Dunno Dunno
+POIN Tutorial
+POIN TrapData TrapData
+POIN Units Units
+POIN $0 $0 $0 $0 $0 $0
+POIN BeginningScene EndingScene
+
+Units:
+UNIT Eirika EphraimMasterLord 0x00 Level(20, Ally, 1) [15,28] 0x00 0x00 0x0 0x00 [Reginleif] NoAI 
+UNIT Seth Swordmaster 0x00 Level(20, Ally, 1) [14,29] 0x00 0x00 0x0 0x00 [SilverSword] NoAI 
+UNIT Gilliam Paladin 0x00 Level(20, Ally, 1) [15,29] 0x00 0x00 0x0 0x00 [SilverLance] NoAI 
+UNIT Ross Paladin 0x00 Level(20, Ally, 1) [14,28] 0x00 0x00 0x0 0x00 [Spear] NoAI 
+UNIT Colm Sniper 0x00 Level(20, Ally, 1) [15,27] 0x00 0x00 0x0 0x00 [SilverBow] NoAI 
+UNIT Vanessa Bishop 0x00 Level(20, Ally, 1) [12,29] 0x00 0x00 0x0 0x00 [Physic, Lightning, Warp, Rescue] NoAI 
+UNIT Lute WyvernLord 0x00 Level(20, Ally, 1) [17,28] 0x00 0x00 0x0 0x00 [SilverLance, Spear, DelphiShield, ChestKey] NoAI 
+UNIT Vanessa General 0x00 Level(20, Ally, 1) [16,27] 0x00 0x00 0x0 0x00 [SilverAxe] NoAI 
+UNIT Neimi Rogue 0x00 Level(20, Ally, 1) [17,27] 0x00 0x00 0x0 0x00 [KillingEdge] NoAI 
+UNIT Artur Hero 0x00 Level(20, Ally, 1) [16,28] 0x00 0x00 0x0 0x00 [HandAxe, Armourslayer] NoAI 
+UNIT Garcia Sage 0x00 Level(20, Ally, 1) [16,29] 0x00 0x00 0x0 0x00 [Elfire] NoAI 
+UNIT Natasha Warrior 0x00 Level(20, Ally, 1) [17,29] 0x00 0x00 0x0 0x00 [SilverAxe] NoAI 
+UNIT
+
+
+TurnBasedEvents:
+TurnEventPlayer(0x0,EventAfterExitingPrepScreen,1)
+END_MAIN
+
+CharacterBasedEvents:
+CharacterEventBothWays(0x9,KojimaGet,Kojima,KojimaNPC)
+CharacterEventBothWays(0x9,SakaguchiGet,Sakaguchi,SakaguchiNPC)
+CharacterEventBothWays(0x9,IgaGet,Iga,IgaNPC)
+END_MAIN
+
+LocationBasedEvents:
+House(0x0,EndingScene,20,18) //Visit the throne and the chapter is complete. A mock arrive, so to say.
+Village(0x7,ChannelingXavier,29,5)
+Village(0x8,SilverAxeVillage,2,25)
+Chest(Audhulma,12,11) //the Chest of Loss, located in that random chest formation
+//give it some fancy effect later
+Armory(Ch28Armoury,21,14)
+Vendor(Ch28Vendor,22,13)
+Village(0xA,RaiseBridge,10,27)
+END_MAIN
+
+MiscBasedEvents:
+CauseGameOverIfLordDies
+END_MAIN
+
+Dunno:
+//DO NOT TOUCH
+WORD $00
+
+Tutorial:
+//DO NOT TOUCH
+WORD $00
+
+TrapData:
+BLST [17,7] IronBallista
+BLST [18,8] IronBallista
+BLST [18,22] Ballista
+END_MAIN
+
+ALIGN 4
+
+BeginningScene:
+
+LOAD1 0x1 MetalGear
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 FinalFantasy
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 Castlevania1
+ENUN
+LOAD1 0x1 SimonsQuest
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 Units
+ENUN
+Text(PlaceholderText)
+//LOAD1 0x1 FillerGuy
+ENDB
+
+EndingScene:
+Text(PlaceholderText)
+CHECK_EXISTS KojimaNPC
+BEQ  0x1 0xC 0x0
+	TurnNPC(KojimaNPC)
+	TurnNPC(SakaguchiNPC)
+	TurnNPC(IgaNPC)
+GOTO 0x2
+LABEL 0x1
```

Comment:
This label is just a GOTO 0x2. Superfluous or msising events?

Reply:
If I understand my conditonals right, the label serves to jump over the TurnNPC codes. The game crashes if they aren't loaded otherwise.

Reply:
I mean you could make the events
```
BEQ  0x1 0xC 0x0
	TurnNPC(KojimaNPC)
	TurnNPC(SakaguchiNPC)
	TurnNPC(IgaNPC)
LABEL 0x1
MNCH 29
ENDA
```
The current extra label and goto are superfluous.